onward/app/controllers/StoryPackageController.scala

```diff
@@ -24,7 +26,7 @@ class StoryPackageController(val contentApiClient: ContentApiClient, val control
     for {
       items <- resp.map(item => StoryPackages(path, item).items)
     } yield {
-      Cached(5.minutes) {
+      val json = if (request.forceDCR) {
```

Comment:
I just wanted to check that this condition is correct? So the implementation we want for DCR _is_ the implementation which was already here? I was expecting that the condition would be `!request.forceDCR`, but I'm probably just missing context.

Reply:
Nah you're right - this is the wrong way round. Thanks! I've made a fix.